wiki/Ways-Of-Working.md

```diff
@@ -1,21 +1,104 @@
+
+Work _should not_ begin on a ticket until (1) an engineer has been assigned to the ticket (2) it has been assigned a [story-point value](#story-point-estimation) (3) it has been assigned an "In Progress" status label.
+
+Typically, engineers _should_ only be working on one "In Progress" ticket at a time. If blocked, apply the "blocked" project label and move on to next Teed-Up item (which should now be marked as "In Progress"). 
+
+### Blockers
+
+The "blocked" label is the official source of truth as to whether a story is considered blocked. Unless it has such a label, a story should not be considered blocked or referenced as blocked. This is to prevent miscommunication across the team. 
+
+### Story Point Estimation
+
+1. You *must* add point estimations to your stories before moving them to "In Progress." (Even better if you can have stories estimated while "Teed Up.")
+2. We use Fibonacci points, i.e. 1, 2, 3, 5, 8, 13. (See below for full breakdown of point values.)
+3. Stories of any point value *may* be broken out into subtasks.
+4. Stories of more than 3 points *must* be broken out into subtasks.
+5. Stories may be broken into _independent_ and _dependent_ sub-tasks, and should be flagged accordingly, to prevent blocked work.
+6. When you complete the work, if the original estimation was off, do not mutate it in place. Instead, add the correct estimation as a comment to the issue with a short explanation. 
+
+NB. Points != Time! Points are an estimate of complexity, not time. Having said that, complexity may roughly map to time, IF there are no other dependencies:
+- 1 point ~= 1-2 hour task
+- 2 points ~= 1/2 day task
+- 3 points ~= all-day task
+- 5 points ~= multi-day task
+- 8 points ~= 1 sprint / 1-week task
+- 13 points ~= more than a single sprint
+
+Points _may_ always be increased mid-implementation, but a justification _must_ be provided.
+
+## Pull Requests
+
+PRs should always reference their instigating ticket in the description, by number.
+
+Engineers are welcome to open several pull requests to close an outstanding ticket. Such PRs can be either "dependent" (i.e., blocking) or "sequential" (i.e., logically independent), and should be flagged accordingly, so that depended-upon (i.e. blocking) PRs are prioritized for merging. 
 
 Close your own PR: if it's ready for QA, tag a reviewer, complain in a Slack channel, or send a DM. 
 
-Never change the title or description of someone else's Github story without permission. 
+### Github Quality Checks
+
+Pull requests to Master _should_ pass Github's code quality checks. (Nb: Not the test suite run by our CI, but those quality checks identified in the PR's `files/diff`.) PR authors _should_ resolve these checks themselves.
+
+Any pull requests originating in the experimental branch, and being merged into Master, _must_ pass all code quality checks. PR authors _must_ resolve these checks themselves.
```

Comment:
Where did this line come from?

Reply:
@jnaviask This was requested by Forest as an addition to the docs [here](https://github.com/hicommonwealth/commonwealth/pull/4770#issuecomment-1677859943):

> for anything coming off the experimental branch into master, we really need it to pass all code quality checks. Meaning, not the test suite run by our CI/CD (which we informally refer to as our CI) but the code quality checks Github identifies in the [files/diff](https://github.com/hicommonwealth/commonwealth/pull/4770/files) of the PR.
> NB. These checks will be code that was not changed in the PR itself, but other instances of code quality issues in files that are touched by the PR. Our stated policy is that these quality checks should be resolved by the author, for core. For the experimental branch, these code quality checks must be resolved.